reflex/components/radix/themes/components/text_field.py

```diff
@@ -79,10 +83,79 @@ def create(cls, *children, **props) -> Component:
         Returns:
             The component.
         """
+        component = super().create(*children, **props)
         if props.get("value") is not None and props.get("on_change"):
             # create a debounced input if the user requests full control to avoid typing jank
-            return DebounceInput.create(super().create(*children, **props))
-        return super().create(*children, **props)
+            return DebounceInput.create(component)
+        return component
+
+    @classmethod
+    def create_root_deprecated(cls, *children, **props) -> Component:
+        """Create a Fragment component (wrapper for deprecated name).
+
+        Copy the attributes that were previously defined on TextFieldRoot in 0.4.9 to
+        any child input elements (via custom_attrs).
+
+        Args:
+            *children: The children of the component.
+            **props: The properties of the component.
+
+        Returns:
+            The component.
+        """
+        console.deprecate(
+            feature_name="rx.input.root",
+            reason="use rx.input without the .root suffix",
+            deprecation_version="0.5.0",
+            removal_version="0.6.0",
+        )
+        inputs = [
+            child
+            for child in children
+            if isinstance(child, (TextFieldRoot, DebounceInput))
+        ]
+        slots = [child for child in children if isinstance(child, TextFieldSlot)]
+        carry_props = {
```

Comment:
I think the event handlers may also need to be bubbled (based on the integration tests)

Reply:
hmm, i'm actually surprised that code sample in reflex-web actually worked before... i thought a root always needed an explicit input inside of it, but i guess not. will adjust